types/mixpanel-browser/index.d.ts

```diff
@@ -100,55 +107,82 @@ export type Response = VerboseResponse | NormalResponse;
-  get_config(prop_name?: string): any;
-  get_distinct_id(): any;
-  get_property(property_name: string): any;
-  has_opted_in_tracking(options?: Partial<HasOptedInOutOptions>): boolean;
-  has_opted_out_tracking(options?: Partial<HasOptedInOutOptions>): boolean;
-  identify(unique_id?: string): any;
-  init(token: string, config?: Partial<Config>, name?: string): Mixpanel;
-  opt_in_tracking(options?: Partial<InTrackingOptions>): void;
-  opt_out_tracking(options?: Partial<OutTrackingOptions>): void;
-  push(item: PushItem): void;
-  register(props: Dict, days?: number): void;
-  register_once(props: Dict, default_value?: any, days?: number): void;
-  reset(): void;
-  set_config(config: Partial<Config>): void;
-  time_event(event_name: string): void;
-  track(event_name: string, properties?: Dict, optionsOrCallback?: RequestOptions | Callback, callback?: Callback): void;
-  track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
-  track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
-  unregister(property: string): void;
-  people: People;
+    add_group(group_key: string, group_id: string, callback?: Callback): void;
+    alias(alias: string, original?: string): void;
+    clear_opt_in_out_tracking(options?: Partial<ClearOptOutInOutOptions>): void;
+    disable(events?: string[]): void;
+    get_config(prop_name?: string): any;
+    get_distinct_id(): any;
+    get_group(group_key: string, group_id: string): Group;
+    get_property(property_name: string): any;
+    has_opted_in_tracking(options?: Partial<HasOptedInOutOptions>): boolean;
+    has_opted_out_tracking(options?: Partial<HasOptedInOutOptions>): boolean;
+    identify(unique_id?: string): any;
+    init(token: string, config?: Partial<Config>, name?: string): Mixpanel;
+    opt_in_tracking(options?: Partial<InTrackingOptions>): void;
+    opt_out_tracking(options?: Partial<OutTrackingOptions>): void;
+    push(item: PushItem): void;
+    register(props: Dict, days?: number): void;
+    register_once(props: Dict, default_value?: any, days?: number): void;
+    remove_group(group_key: string, group_ids: string | string[] | number | number[], callback?: Callback): void;
+    reset(): void;
+    set_config(config: Partial<Config>): void;
+    set_group(group_key: string, group_ids: string | string[] | number | number[], callback?: Callback): void;
+    time_event(event_name: string): void;
+    track(
+        event_name: string,
+        properties?: Dict,
+        optionsOrCallback?: RequestOptions | Callback,
+        callback?: Callback,
+    ): void;
+    track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
+    track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
+    unregister(property: string): void;
+    people: People;
 }
 
+export function add_group(group_key: string, group_id: string, callback?: Callback): void;
```

Comment:
@karlos1337 why do we have individual function exports as well as [the ones defined in the `Mixpanel` interface](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46097/files#diff-55d6f62151213aea3e8e542d44ee24c3R144)?

Reply:
Sorry @dwilt, can't say, I'm totally disconnected from this, I'd created this definitions for a project that I'm not currently maintaining and from so long 😅, maybe @RicardoRodrigues or @randak  could help.

Reply:
Unfortunately, I've only recently started working on the types for this library, because we use it regularly in my work. I don't have any historical context on why the definitions are structured this way. If you can test and verify whether the typing is valid without the duplication, perhaps you would be willing to create a follow up PR after this one is merged in order to make that change?

Reply:
@karlos1337 You may want to remove yourself from the maintainers list on these definitions so you don't keep getting tagged with this 😄

Reply:
I might look at firing up another PR to look at this duplicate issue.